pyproject.toml

```diff
@@ -26,10 +26,9 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
 ]
 dependencies = [
-  'astropy>=4.1.0',
-  'numpy>=1.16.0',
   'hermes_core @ git+https://github.com/HERMES-SOC/hermes_core/',
-  'ccsdspy @ git+https://github.com/ddasilva/ccsdspy.git'
+  'ccsdspy @ git+https://github.com/ddasilva/ccsdspy.git',
+  'ndcube>=2.1.3', # Temporary while not part of hermes_core
```

Comment:
Temporarily aded this line while https://github.com/HERMES-SOC/hermes_core/pull/93 is open. Once the PR in Core completes, ndcube will be installed with the `hermes_core` dependency.